internal/auth0/action/resource_trigger_action.go

```diff
@@ -86,21 +94,20 @@ func createTriggerAction(_ context.Context, d *schema.ResourceData, m interface{
 		})
 	}
 
-	action, err := api.Action.Read(actionID)
-	if err != nil {
-		if mErr, ok := err.(management.Error); ok && mErr.Status() == http.StatusNotFound {
-			d.SetId("")
-			return nil
+	if d.GetRawConfig().GetAttr("display_name").IsNull() {
```

Comment:
Because the `display_name` is computed and is of type string it will never actually be `nil`, plus we're already having this value from line 70. This should be changed to:

```suggestion
	if displayName == "" {
```